.github/workflows/release.yml

```diff
@@ -54,8 +54,7 @@ jobs:
       run: ./gradlew build
     - name: Gradle Publish
       env:
+        RELEASE_VERSION: ${{ github.ref_name }}
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         GITHUB_ACTOR: ${{ secrets.GITHUB_ACTOR }}
-      run: ./gradlew publish
-    - name: Increment version
-      run: ./gradlew incrementBuildVersion
+      run: ./gradlew publish
```

Comment:
Do we still need this if we are publishing in post-release?

Reply:
Yes because even though the publish has happened for the package we still need to publish the kotlin sdk which happens in this gradle task https://github.com/xmtp/proto/blob/main/kotlin/build.gradle#L64-L87 This is just calling this gradle task.

Reply:
O shoot now I see where your comment was I thought I had removed this entirely from the release.yml but yes you're right this entire kotlin block should be removed from here.